Network/Socket.hs

```diff
@@ -272,9 +405,10 @@ import Network.Socket.Info
 import Network.Socket.Internal
 import Network.Socket.Name hiding (getPeerName, getSocketName)
 import Network.Socket.Options
+import Network.Socket.STM
 import Network.Socket.Shutdown
 import Network.Socket.SockAddr
-import Network.Socket.Syscall hiding (connect, bind, accept)
+import Network.Socket.Syscall hiding (accept, bind, connect)
```

Comment:
Is this file formatting only?

Reply:
Yeah. Sorry. `fourmolue` did it automatically.